test/Microsoft.TestPlatform.AcceptanceTests/CodeCoverageTests.cs

```diff
@@ -247,7 +246,7 @@ private void CollectCodeCoverage(RunnerInfo runnerInfo, TestParameters testParam
     }
 
     private string CreateArguments(
-        TempDirectory tempDir,
+        TempDirectory TempDirectory,
```

Comment:
I don't think you want this rename. Also, do we want to give the possibility to pass a temp dir or shall we use the base property directly within the method?

Reply:
I reverted the rename. How temp directory is passed is up the to test author, I am not trying to change everything in this PR.